docs/src/_includes/layouts/doc.html

```diff
@@ -88,7 +88,7 @@ <h1>{{ title }}</h1>
                     index: id,
                     recommended: rule_meta.docs.recommended,
                     fixable: rule_meta.fixable,
-                    frozen: rule_meta.docs.frozen,
+                    frozen: false if rule_meta.deprecated else rule_meta.docs.frozen,
```

Comment:
I'm not sure if this is necessary, because when we deprecate a rule, I think we should remove `frozen: true` from its `meta.docs`.

Reply:
In other words, rule shouldn't be marked as deprecated and frozen at the same time. Frozen means we're not accepting new feature requests, but we still fix bugs. Deprecated means we're not accepting any changes.

Reply:
Okay, then this PR doesn't make sense.